app/views/candidate_interface/unsubmitted_application_form/_adviser_call_to_action.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if adviser_sign_up.available? %>
```

Comment:
Should we conditionally render the partial rather than conditionally rendering its contents?

Reply:
I had it that way originally but felt the check should be encapsulated in the partial; my reasoning being if the partial was included elsewhere we would have to duplicate the availability check in the view if it was outwith it. I don't feel strongly on this either way though, so if you do I'm happy to change it.